js/table/tablemanager.ts

```diff
@@ -21,6 +21,13 @@ import { TournamentView } from "./tournamentview";
 declare var apiHost: string;
 declare var host: string;
 
+export enum CardsDealedCodes {
```

Comment:
I'm not shure if this enum should be in tables manager or in anouther file with models